docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,152 @@ metadata:
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-platform-pool.drawio.svg)
+
+---
+
+It is possible to extend the configuration rule entry with region specified. Below configuration means that if a cluster is provisioned in cf-sa30 subaccount regiont then KEB would search for secret bindings with `hyperscalerType: gcp_cf-sa30` label. In other cases, for example when probisioning a gcp cluster in other region, KEB would search for `hyperscalerType: gcp` label. Differently than in previous example, in this case, it is possible to configure two different pools for the same hyperscaler type.
+
+```
+hap: 
+  platformRegionRule: "gcp:cf-sa30"
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-platform-region-pool.drawio.svg)
+
+---
+
+As stated previously, it is possible to include multiple elements in each configuration rule. The example below shows how to configure multiple plan/region pairs for based on platformRegionRule. The same can be configured for all other rules (clusterRegionRule, sharedRule, euAccessRule). The list below allows to KEB to operate on 3 pools for gcp plan: one withouth a region, one for cf-sa30 region and one for cf_jp30 region; 2 pools for aws plan: one without a region and one for cf_eu11 region; multiple pools for openstack plan - all with a platform region specified; and one pool for azure without a platform region.
+
+```
+hap: 
+  platformRegionRule: "gcp:cf-sa30;gcp:cf-jp30;aws:cf-eu11;openstack"
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-platform-region-pool-list.drawio.svg)
+
+---
+
+Below configuration specifies azure plan as one for which to use cluster region based search. In this case, cluster provisioned in with the azure plan would require a secret binding with `hyperscalerType: azure_<CLUSTER_REGION>` label. All of the rule examples apply all of platformRegionRule, clusterRegionRule, shareRule and euAccessRule properties.
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: "azure"
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-azure-pool.drawio.svg)
+
+---
+
+All the configuration rules are independent of each other from the point of view of their configuration. If the same plan/region pair appears in more than one property, then all the rules that it appears in take effect. Below configuration specifies that for all gcp clusters the search should be done based on platform region and cluster region searches. The resulting search label would be `hyperscalerType: gcp_<PLATFORM_REGION>_<CLUSTER_REGION>`.
```

Comment:
Is `property` defined somewhere, or this is synonymous with rule?

Reply:
And I guess pairs plan/region could be used only in sharedRule and euAccessRule.